include/exadg/operators/inverse_mass_operator.h

```diff
@@ -91,9 +109,36 @@ class InverseMassOperator
     }
   }
 
+  void
+  initialize_inverse_mass_operator_with_block_jacobi()
+  {
+    // initialize mass operator
+    dealii::AffineConstraints<Number> const & constraint =
+      matrix_free->get_affine_constraints(dof_index);
+
+    MassOperatorData<dim> mass_operator_data;
+    mass_operator_data.dof_index  = dof_index;
+    mass_operator_data.quad_index = quad_index;
+
+    mass_operator.initialize(*matrix_free, constraint, mass_operator_data);
+
+    // build a BlockJacobiPreconditioner and use the vmult(dst,src) for applying the inverse mass
+    // operator on  source the vector
+    mass_preconditioner =
+      std::make_shared<BlockJacobiPreconditioner<MassOperator<dim, n_components, Number>>>(
+        mass_operator);
+  }
+
   dealii::MatrixFree<dim, Number> const * matrix_free;
 
   unsigned int dof_index, quad_index;
+
+  // BlockJacobi Preconditoner to be used when the MatrixFreeInverseMass is not available
```

Comment:
I think this comment belongs to line 141 (preconditioner) below. I would like to avoid the negation in "is not available" here, again for code readability. Make a positive statement like "For special cases, deal.II provides fast matrix-free routines to invert the mass operator in an explicit way".